src/browser/provider/phantomjs.js

```diff
@@ -0,0 +1,47 @@
+import { create as createPhantom } from 'phantom';
+
+export default {
```

Comment:
It doesn't implement list available browsers, what `testcafe -b phantomjs` will show?